.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+jordan.xcuserdatad
+/JordanResources
```

Comment:
Thinking instead of committing this, lets use the [official gitignore](https://github.com/github/gitignore/blob/master/Objective-C.gitignore) what do you think? That way we future proof against 'all' possibilities?